packages/charts/react-charting/src/components/DeclarativeChart/PlotlySchemaAdapter.ts

```diff
@@ -3,6 +3,8 @@
 /* eslint-disable no-var */
 /* eslint-disable no-console */
 /* eslint-disable @typescript-eslint/no-explicit-any */
+import * as React from 'react';
```

Comment:
Move the state map to DeclarativeChart. 
Schema adapter is supposed to be stateless.

Reply:
done